doc/radosgw/s3-notification-compatibility.rst

```diff
@@ -51,6 +51,8 @@ Ceph's bucket notification API has the following extensions:
 
   - Filtering based on object tags
 
+- Each one of the additional filters extends the S3 API and using it will cause extension of the client SDK as well (unless you are using plain HTTP). 
```

Comment:
this is not clear. maybe you mean:
Each one of the additional filters extends the S3 API and using it will require extension of the client SDK.

Reply:
Changed it.